confirm.c

```diff
@@ -763,16 +765,18 @@ confirm_query_create4(const struct sockaddr_storage *dst, uint8_t ttl,
 }
 
 struct confirm_query *
-confirm_query_create6(const struct sockaddr_storage *dst, uint8_t ttl,
-		uint8_t traffic_class, uint32_t flow_label,
-		uint16_t icmpid, uint8_t flowid,
-		confirm_query_cb cb)
+confirm_query_create6_icmp(const struct sockaddr_storage *dst, uint8_t ttl,
+		uint8_t traffic_class, uint32_t flow_label, uint16_t icmpid,
+		uint8_t flowid, confirm_query_cb cb)
 {
 	struct confirm_query *query;
-
-	query = malloc(sizeof(struct confirm_query));
+	query = malloc(sizeof(*query));
 	if(!query) logea(__FILE__, __LINE__, NULL);
 
+	if(flowid > CONFIRM_MAX_FLOWID) {
+		logd(LOG_WARN, "%s,%d: flowid > 127!\n", __FILE__, __LINE__);
```

Comment:
This is a bit confusing.  If I remember correctly, there is this `MAX_FLOWID` limit simply because I had a 128-position vector of random 16-bit numbers to use as flow identifiers. I understand why we want the flowid to be less than the vector length (128 positions), otherwise we would have an array out-of-bounds erro.r

The issue of how many probes we allow in parallel seems to be separate. Why do we need to limit the number of concurrent probes? As long as we have enough identifiers and can map responses (time exceeded or port unreachable) to probes, there is no reason to limit the number of concurrent probes, right?  (See, e.g., Yarpping.)

BTW, how do we differentiate between probes of multiple mda6 instances (e.g., when we run multiple instances concurrently in Linode)?